### PR TITLE
Plugin name and contributing instructions

### PR DESCRIPTION
**Pull request by pdewouters**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,9 +11,17 @@ Develop on a feature branch and send a pull request to BackUpWordPress' master b
...
 ## Translations ##
 
 Please don't submit pull requests for translations. All translations are managed via http://translate.hmn.md
 Please request access via email at hello@hmn.md, mentioning "glotpress access request" in the subject line.
 
 Thank you for contributing!
+
+## Weekly dev chat ##
+
+Request to be added as a member to our Slack channel: support@hmn.md
```
Comment (willmot) [inline CONTRIBUTING.md]: I think we should be a bit more verbose and welcoming here.

Also shouldn't the Slack channel be `https://backupwordpress.slack.com/`?

**Comment by willmot** [comment]
cc @pdewouters @owaincuvelier would love to here your thoughts on my rewrite of the contribution guidelines. Happy to merge in?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,19 +1,46 @@
...
+- Developing
+- Testing
+- Translating
 
-## Workflow ##
-Develop on a feature branch and send a pull request to BackUpWordPress' master branch for review.
+## Developing
 
-Assign the pull request to Paul de Wouters: [@pdewouters](https://github.com/pdewouters).
+Develop your changes a feature branch and send a pull request to BackUpWordPress' `master` branch for review.
```
Comment (pdewouters) [inline CONTRIBUTING.md]: Develop your changes **in** a feature

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,19 +1,46 @@
...
+
+## Translating
+
+We want BackUpWordPress to be available in as many languages as possible.
+
+All translations are managed via http://translate.hmn.md/ please contribute there rather than by submitting new translation files here. Please request access via email at support@hmn.md, mentioning "glotpress access request" in the subject line.
+
+## Weekly Dev Chat
+
+We have a weekly dev chat in [Slack](https://backupwordpress.slack.com) and we'd love you to join. We use the chat to run through new issues that have come in, discuss any open Pull Requests and talk through the plans and progress for the next release.
```
Comment (pdewouters) [inline CONTRIBUTING.md]: Don't they need to request to be added to the slack channel somehow?

**Comment by willmot** [comment]
Cheers @pdewouters, have updated. @owaincuvelier going to merge but please do chime in with thoughts / changes if you notice anything 👍

**Pull request merged**